src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -2302,12 +2435,12 @@ fn test_string_ascii_fold() {
     let good = [
         "(define-private (get-len (x (string-ascii 1)) (acc uint)) (+ acc u1))
         (fold get-len \"blockstack\" u0)",
-        "(define-private (slice (x (string-ascii 1)) (acc (tuple (limit uint) (cursor uint) (data (string-ascii 10)))))
+        "(define-private (slice-v1 (x (string-ascii 1)) (acc (tuple (limit uint) (cursor uint) (data (string-ascii 10)))))
```

Comment:
are you renaming these to avoid a conflict?

maybe rather than renaming these, you can run these old contracts in Clarity1?

Reply:
The point of this test was not to test across clarity versions - it is a coincidence that we had an old test that defined a function with the name `slice`, so I think renaming it is the most clear thing to do.